src/megatron/energon/task_encoder/cooking.py

```diff
@@ -53,4 +54,8 @@ def basic_sample_keys(crude_sample: dict) -> dict:
     """A convenience helper to extract the basic keys from a crude sample,
```

Comment:
So we still have basic_sample_keys? I though that'd be replaced by `derive_from`.
If we want to keep it for backwards compatibility, we should still adapt the cooking docs.

Reply:
That's different, because `basic_sample_keys` adresses dicts, while `derive_from` adresses `Sample`s